src/api/types.ts

```diff
@@ -0,0 +1,15 @@
+interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
```

Comment:
You can define a Todo interface like so without additional extensions
```suggestion
interface Todo {
  id: number,
  title: string,
  completed: boolean,
  userId: number,
  user?: User,
}
```